packages/codecs/src/feed.ts

```diff
@@ -15,6 +15,7 @@ export const JsonAsString = new Type<unknown, string, string>(
 )
 
 export const AggregationDocument = type({
+  token: string,
```

Comment:
should we call this `resumeToken` or something else to make it clearer what it's used for?

Reply:
Let it be resumeToken.